client/widgets/RadialMenu.cpp

```diff
@@ -21,14 +21,14 @@
 
 #include "../../lib/CGeneralTextHandler.h"
 
-RadialMenuItem::RadialMenuItem(const std::string & imageName, const std::string & hoverText, const std::function<void()> & callback)
+RadialMenuItem::RadialMenuItem(const std::string & imageName, const std::string & hoverText, const std::function<void()> & callback, bool alternativeLayout)
 	: callback(callback)
 	, hoverText(hoverText)
 {
 	OBJ_CONSTRUCTION_CAPTURING_ALL_NO_DISPOSE;
 
-	inactiveImage = std::make_shared<CPicture>(ImagePath::builtin("radialMenu/itemInactive"), Point(0, 0));
-	selectedImage = std::make_shared<CPicture>(ImagePath::builtin("radialMenu/itemEmpty"), Point(0, 0));
+	inactiveImage = std::make_shared<CPicture>(ImagePath::builtin(alternativeLayout ? "radialMenu/itemInactiveAlt" : "radialMenu/itemInactive"), Point(0, 0));
+	selectedImage = std::make_shared<CPicture>(ImagePath::builtin(alternativeLayout ? "radialMenu/itemEmptyAlt" : "radialMenu/itemEmpty"), Point(0, 0));
```

Comment:
Why different layout is needed?

Reply:
It's a 90 degrees rotated layout.

With this it's possible to have two vertical buttons one below the other.

Reply:
But it is not really vertical from screenshots:
![grafik](https://github.com/vcmi/vcmi/assets/13953785/008a5899-e17c-418a-94e1-76f6a5b4ed76)

Reply:
And does non-rotated version actually looks bad?

Reply:
But the down button is exactly under the up button (without space). That's not possible with non rotated layout.